roles/tls-cert/license.md

```diff
@@ -18,11 +17,11 @@ met:
  * Redistributions in binary form must reproduce the above copyright
    notice, this list of conditions and the following disclaimer in the
    documentation and/or other materials provided with the distribution.
- * Neither the name of the iPlant Collaborative, BIO5, The University
-   of Arizona, Cold Spring Harbor Laboratory, The University of Texas at
-   Austin, nor the names of other contributors may be used to endorse or
-   promote products derived from this software without specific prior
-   written permission.
+ * Neither the name of the CyVerse, BIO5, The University of Arizona, 
```

Comment:
👍

Reply:
s/the CyVerse/CyVerse

Reply:
That's something to fix in [ansible-tls-cert](https://github.com/CyVerse-Ansible/ansible-tls-cert/commits/master), which can then be reinstalled here